Kickstarter-iOS/Views/CommentInputContainerView.swift

```diff
@@ -20,12 +19,13 @@ final class CommentInputContainerView: UIView {
     CommentInputTextView(frame: .zero)
   }()
 
-  let placeholderLabel: UILabel = { UILabel(frame: .zero) }()
   lazy var postButton: UIButton = {
     UIButton(type: .system)
       |> \.isHidden .~ true
   }()
 
+  let placeholderLabel: UILabel = { UILabel(frame: .zero) }()
+  private let postButtonTitleFont = UIFont.systemFont(ofSize: 15, weight: .semibold)
```

Comment:
For buttons we actually don't want the font size to increase with accessibility font size changes, it becomes problematic fast so we typically keep it at a fixed size just for buttons.

Reply:
Trying to recall the back-story here 😄

Reply:
Ah good call, so I updated to `ksr_headline` which is 17 point semi-bold, it'll be replacing 15 point semi bold, hopefully design is cool with that.

Reply:
Oh ok, in response to @justinswart I'll leave this as-is.

Reply:
Yeah but I'm also trying to remember why this is. For example we once had buttons resizing to accessibility sizes, we also had multiline buttons and we removed all of that. So I'm not sure if we want the buttons to wrap with `numberOfLines = 0` 🤔 Let me see if I can find the older PR's about this.

Reply:
Ok, looks like we did it because of the styles of the buttons we were using - https://github.com/kickstarter/ios-oss/pull/806. Given that the post button is a _text_ type button it is probably ok to do it here 👍

Reply:
ok, I'll address this now then using `ksr_headline` as a replacement.

Reply:
Oh wait, probably best to address this when we pair...just to be sure I'm not misunderstanding the intent of making the buttons `preferredFont`